### ISSUE TITLE
Skewer did not go where I cast it

### ISSUE THREAD

**Issue by shawn-simon** [ValveSoftware/Dota2-Gameplay#9902]
### Description

I have looked at this in the game for a while, I'm pretty confident that my mouse was not on the minimap at cast time, despite what folks in the thread say:

https://www.reddit.com/r/DotA2/comments/14jay18/game_losing_skewer_bug_anyone_know_what_happened/

And if it is registering a minimap click, that doesn't seem great either, based on the position of my mouse at the time and the fact no one would ever want to use the minimap this way. I also have minimap accidental click protection set to the max (.5s) but I am guessing that doesn't apply to quickcast.

### Example Match ID (and possibly Timestamp)

7216383741 41:31

### Screenshots

Cursor location of skewer: https://steamcommunity.com/sharedfiles/filedetails/?id=2994867361
Video: https://www.reddit.com/r/DotA2/comments/14jay18/game_losing_skewer_bug_anyone_know_what_happened/

**Comment by cadaeic** [comment]
You pointed the cursor at the minimap momentarily at 0:03.

**Comment by commentator24** [comment]
you started skewer at the border of minimap (according to my HUD)
playing at 0.25 speed this is the actual location of your cursor. you had the wrong screenshot due to frame and replay speed limitations.
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/0e925e1d-ce13-4acb-a429-67b581a20fb1)
also tested in lobby, clicking this spot will zoom camera there and yielded the same result.
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/c8a2fbdb-f835-4e46-b58b-b890f6fdfde7)

I would agree minimap misclick protection time is not really working.
i guess you can switch the location of minimap to right side to prevent accidental clicks.
using skills on minimap is a nice feature to have on other heroes not just magnus, e.g. playing as NP using ult start from top / bottom side, as treant living armor on hero that looks like having a teamfight at top.
also, you RP too early, shouldve wait for treant 6 seconds respawn. let T3 go, ult skewer into treant ult wouldve finished them up.

**Comment by cadaeic** [comment]
I agree that it would be a nice QoL change to disable minimap targeting for most skills. Nobody is going to cast Light Strike Array on the minimap.

Skewer is however one of the few exceptions. When you need to escape immediate danger you can target the minimap to Skewer yourself up to 1500 range away without having to move the camera. If the minimap gets in your way too often, perhaps you could consider the right-side minimap.

**Comment by commentator24** [comment]
did you forget that 7.33 is a bigger map so minimap scaling perhaps not accurate because of HUD overlay (e.g. my wood themed border stays the same). if minimap includes inaccessible area it would look weird too and make minimap look tiny.

I'd assume it's targetable but not accessible for several reasons:
1. to center your screen at the edges of map. otherwise little room to click / view the edges of map / use abilities / change camera perspective e.g. from trees blocking view when you dropped a gem etc.
2. aesthetically better than simply black area.
3. coding / engine limitations. otherwise out of boundary / more issues with camera perspective / HUD maybe.
4. what solution do you propose that would make all players happy? if minimap bigger there will be more accidents.. if the border can't be clicked that might be nice though might be a little annoying to some players
5. maybe a way for flying couriers to hide from minimap? you're right that it's about 3x dire fountain away as shown
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/d8c827b2-1b95-4a6f-a497-276d92fa3ff0)
